perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -394,6 +395,36 @@ sub direct_debit_renew : Path('dd_renew') : Args(0) {
     $c->res->body('ERROR - DD renewal is automatic');
 }
 
+sub csc_code : Path('csc_code') : Args(0) {
```

Comment:
Presumably this should be publicly viewable at a path?

Reply:
Do you mean should not? probably not, no.